### DISCUSSION TITLE
Hall of Fame: Share your experience with apalis 🚀

### DISCUSSION THREAD

**Discussion by geofmureithi** [apalis-dev/apalis#272] [Show and tell]
A lot of work has gone into working on this library. I am aware of a good number of rustaceans using this crate.
Take a moment to share what you are doing and your experience with apalis.

**Comment by geofmureithi** [comment]
I will go first. Here are some projects built with apalis.
- [Ryot](https://github.com/IgnisDa/ryot): A self hosted platform for tracking various facets of your life - media, fitness etc.
- [Summarizer](https://github.com/akhildevelops/summarizer): Podcast summarizer
- [Universal Inbox](https://github.com/universal-inbox/universal-inbox): Universal Inbox is a solution that centralizes all your notifications and tasks in one place to create a unique inbox.

**Reply by anwar-al-jahwari** [reply]
we are using apalis in Vodafone Oman for Whatsapp Bot Project. it represents the core of the entire bot.
